app/src/js/components/home.js

```diff
@@ -199,63 +220,41 @@ class Home extends React.Component {
-          <section className='page__section datetime'>
-            <div className='row'>
-              <div className='heading__wrapper'>
-                <h2 className='datetime__info heading--medium heading--shared-content--right'>
-                  Select date and time to refine your results. <em>Time is UTC.</em>
-                </h2>
-              </div>
-              <div className='datetime__range_wrapper'>
-                <DatepickerRange onChange={this.query}/>
-              </div>
-            </div>
-          </section>
 
-          <section className='page__section metrics--overview'>
-            <div className='row'>
-              <div className='heading__wrapper--border'>
-                <h2 className='heading--large heading--shared-content--right'>Metrics Overview</h2>
-              </div>
+          {this.pageSection(
+            'datetime', <>Select date and time to refine your results. <em>Time is UTC.</em></>,
+            <div className='datetime__range_wrapper'>
+              <DatepickerRange onChange={this.query}/>
             </div>
-          </section>
+          )}
 
-          {this.renderButtonListSection(overview, 'Updates')}
-          {this.renderDistrubutionErrors(dist)}
-          {this.renderButtonListSection(distErrorStats, 'Distribution Errors', 'distributionErrors')}
-          {this.renderButtonListSection(distSuccessStats, 'Distribution Successes', 'distributionSuccesses')}
+          {this.sectionHeader('metrics--overview', 'Metrics Overview')}
+          {this.buttonListSection(overview, 'Updates')}
 
-          <section className='page__section update--granules'>
-            <div className='row'>
-              <div className='heading__wrapper--border'>
-                <h2 className='heading--large heading--shared-content--right'>Granules Updates</h2>
-                <Link className='link--secondary link--learn-more' to={{ pathname: '/granules', search: searchString }}>{strings.view_granules_overview}</Link>
-              </div>
-            </div>
+          {this.sectionHeader('distribution--overview', 'Distribution Overview')}
+          {this.distrubutionConnectionErrors(dist)}
+          {this.buttonListSection(distErrorStats, 'Distribution Errors', 'distributionErrors')}
+          {this.buttonListSection(distSuccessStats, 'Distribution Successes', 'distributionSuccesses')}
 
-            {this.renderButtonListSection(
-              updated,
-              <>{strings.granules_updated}<span className='num-title'>{numGranules}</span></>
-            )}
+          {this.sectionHeader(
+            'update--granules', 'Granules Updates',
+            <Link className='link--secondary link--learn-more' to={{ pathname: '/granules', search: searchString }}>{strings.view_granules_overview}</Link>
+          )}
+          {this.buttonListSection(
+            updated,
+            <>{strings.granules_updated}<span className='num-title'>{numGranules}</span></>
+          )}
```

Comment:
This is causing an issue in the markup since that jsx header is being passed as the className

`<section class="page__section [object Object]">`